ckanext/depositar_theme/fanstatic/scss/_bootstrap-variables.scss

```
@@ -0,0 +1,12 @@
+$primary: #000;
+$link-decoration: none;
+$link-hover-decoration: none;
+$badge-color: #fff;
+$font-family-base: 'Noto Sans TC', arial, 'Microsoft JhengHei', sans-serif;
```

```suggestion
$font-family-sans-serif: 'Noto Sans TC', 'Arial', 'Microsoft JhengHei', sans-serif;
```
Align with the [upstream](https://github.com/ckan/ckan/blob/ckan-2.10.7/ckan/public/base/scss/_bootstrap-variables.scss#L419).